packages/user-mgnt/src/features/updateUser/handler.ts

```diff
@@ -158,15 +162,19 @@ export default async function updateUser(
   }
 
   if (userNameChanged) {
-    sendUpdateUsernameNotification(
-      user.name,
-      existingUser.username,
-      {
-        Authorization: request.headers.authorization
+    triggerUserEventNotification({
+      event: 'user-updated',
+      payload: {
+        recipient: {
+          name: user.name,
+          email: user.emailForNotification,
+          mobile: user.mobile
+        },
+        username: existingUser.username
       },
-      user.mobile,
-      user.emailForNotification
-    )
+      countryConfigUrl: COUNTRY_CONFIG_URL,
+      authHeader: { Authorization: request.headers.authorization }
+    })
```

Comment:
we should forward the new username to countryconfig instead of calculating it there